.github/troubleshooting.md

```diff
@@ -39,7 +38,7 @@ The Helm charts pipeline has a "release" job that in case of errors, the log sho
 The manual actions to do in case of a failure are:
 
 * Delete the $RELEASE_VERSION-main branch (if exists)
-* Revert the last commit in the $RELEASE_VERSION branch (if there is one made by kiali-bot on the same time around the pipeline run)
```

Comment:
@leandroberetta please review this line. I don't think we should delete this entirely. Perhaps just remove mention of kiali-bot specifically (since kiali-bot user isn't involved now). Perhaps something like this?

```
* Revert the last commit in the $RELEASE_VERSION branch (if there is one made by the GitHub action at the same time around the pipeline run)
```

Reply:
Same here.